pkg/registry/apps/deployment/storage/storage_test.go

```diff
@@ -451,3 +453,123 @@ func TestCategories(t *testing.T) {
 	expected := []string{"all"}
 	registrytest.AssertCategories(t, storage.Deployment, expected)
 }
+
+func TestScalePatchErrors(t *testing.T) {
+	storage, server := newStorage(t)
+	defer server.Terminate(t)
+	validObj := validNewDeployment()
+	resourceStore := storage.Deployment.Store
+	scaleStore := storage.Scale
+
+	defer resourceStore.DestroyFunc()
+	ctx := genericapirequest.WithNamespace(genericapirequest.NewContext(), namespace)
+
+	{
```

Comment:
these closures are not really needed, i guess?
(applies to the other object types's storage / storage_test)

Reply:
I added the blocks to keep the helper function for each step local to avoid copy/paste reference errors